Sources/App/Core/Github.swift

```diff
@@ -97,6 +99,8 @@ extension Github {
         switch resource {
             case .license, .readme:
                 return URI(string: "https://api.github.com/repos/\(trunk)/\(resource.rawValue)\(queryString)")
+            case .funding:
+                return URI(string: "https://api.github.com/repos/\(trunk)/contents/FUNDING.yml")
```

Comment:
I'm pretty sure you've checked this but just in case: this isn't available via GraphQL such we could stick it in the existing call?

Reply:
This information is part of the GraphQL API 😄  Might be a reason not to use it though.

```gql
query {
  repository(owner: "SwiftPackageIndex", name:"SwiftPackageIndex-Server"){
    fundingLinks {
      ... on FundingLink {
        url
        platform
      }
    }
  }
}
```

```json
{
  "data": {
    "repository": {
      "fundingLinks": [
        {
          "url": "https://github.com/SwiftPackageIndex",
          "platform": "GITHUB"
        }
      ]
    }
  }
}
```

Reply:
vapor/vapor - for example

```json
{
  "data": {
    "repository": {
      "fundingLinks": [
        {
          "url": "https://github.com/vapor",
          "platform": "GITHUB"
        },
        {
          "url": "https://opencollective.com/vapor",
          "platform": "OPEN_COLLECTIVE"
        }
      ]
    }
  }
}
```

Reply:
I looked for it in the graphql api before creating this PR and missed that it was there, which is frustrating but just one of those things. 

My original version would have supported the Vapor scenario but the new version from the gql is better, and almost complete